### DISCUSSION TITLE
ATROPISOMERS GRAPHICAL REPRESENTATION

### DISCUSSION THREAD

**Discussion by Alesapo-py** [rdkit/rdkit#8118] [Q&A]
Hello everyone,
I am experiencing issues with the graphical visualization of atropisomers. I would like them to be displayed as shown in the following image: 

![correct_atropoisomers](https://github.com/user-attachments/assets/7072fd8e-6674-4fd2-9454-9ddeaaa01353)

Between the two wedge bonds, there should be a filled bond that maintains continuity in the 3D representation. However, RDKit displays them differently: 

![wrong_atropoisomers](https://github.com/user-attachments/assets/d9a444e6-e9e7-4772-9cd7-49e48c335be3)

As you can see, the bond between atoms 32 and 31 should be shown with a filled line, continuing with the two wedge bonds, but instead, it remains a simple aromatic double bond. Is there a way to represent double bonds with a filled line, like the wedge bonds?

**Comment by DavidACosgrove** [comment]
Hi @Alesapo-py.  I don't think there are right or wrong ways of depicting atropisomers like these.  The IUPAC guidelines (https://publications.iupac.org/pac/pdf/2008/pdf/8002x0277.pdf) have nothing to say on the matter explicitly, although the document does contain examples of the sort of 3-bond wedge you prefer.  These are not in the context of atropisomers, however.  The MarvinSketch documentation (https://docs.chemaxon.com/display/docs/background_axial-stereoisomerism-atropisomerism.md) says that either way can be shown.  Given that an aromatic ring is planar by definition, once one bond is drawn wedged, it is clear to the trained eye that that side of the ring is up or down.  The RDKit choice of representation thus adequately shows the orientation of the ring with the minimum of clutter on the drawing.  This is consistent with the way tetrahedral stereochemistry is shown, where only the first bond of a chain is wedged, for example for atoms 44 and 45 in your larger drawing.

I would be interested to know how the wedging information for your large structure was produced, because I don't think that the bond between atoms 30 and 31 should be wedged.

I suppose we could add the alternative representation as an option, but there are so many drawing options already that we are generally reluctant to add more unless there is a very strong case.

**Reply by Alesapo-py** [reply]
Hi David,
Thanks for the info and your support.
My goal is to represent molecular structures according to the WHO guidelines: https://cdn.who.int/media/docs/default-source/medicines/norms-and-standards/current-projects/2024-07-26_graphicrepresentation-17-726-forpubcon.pdf?sfvrsn=256ffde2_6
While the guidelines don't provide specific instructions for atropisomers, I assume RDKit's representation is acceptable. The extra wedge bond is due to my function: "Retain_Stereo_and_mapping_atoms."
Since I'm working with CDXML files, RDKit doesn't automatically save wedge and dash bonds. So, I've created a function that, based on the bond display parameter in the CDXML file, saves this info in the RDKit molecular object.
Additionally, for some CDXML molecules, RDKit sometimes uses coordinates from nested fragments. To address this, I've developed a remapping function.
